src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/UtentiPGPage.java

```diff
@@ -0,0 +1,349 @@
+            getWebDriverWait(10).withMessage("il titolo della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(titoloAggiungiUtente));
+            getWebDriverWait(10).withMessage("il sottotitolo della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(sottotitoloAggiungiUtente));
+            getWebDriverWait(10).withMessage("il box codice fiscale della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(codiceFiscaleBox));
+            getWebDriverWait(10).withMessage("il box nome della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(nameBox));
+            getWebDriverWait(10).withMessage("il box cognome della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(surnameBox));
+            getWebDriverWait(10).withMessage("il box email della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(emailBox));
+            getWebDriverWait(10).withMessage("il box conferma email della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(confirmEmailBox));
+            getWebDriverWait(10).withMessage("il combobox seleziona prodotto della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(selectProductDropdown));
+            getWebDriverWait(10).withMessage("il bottone indietro della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(indietroButton.get(1)));
+            getWebDriverWait(10).withMessage("il bottone continua della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(continueButton));
+            logger.info("Si visualizza correttamente aggiungi nuovo utente page");
+        }catch (TimeoutException e){
+            logger.error("Non si visualizza correttamente aggiungi nuovo utente con errore:"+e.getMessage());
+            Assert.fail("Non si visualizza correttamente aggiungi nuovo utente page con errore:"+e.getMessage());
+        }
+    }
+
+    public void insertData(String codiceFiscale,String name, String surname, String email ) throws InterruptedException {
+
+        codiceFiscaleBox.sendKeys(codiceFiscale);
+        Thread.sleep(2000);
+        if (nameBox.getAttribute("value").equalsIgnoreCase(name) && surnameBox.getAttribute("value").equalsIgnoreCase(surname)){
+            logger.info("Il nome e il cognome è generato correttamente");
+        }else {
+            logger.error("Il nome e il cognome non è generato correttamente");
+            Assert.fail("Il nome e il cognome non è generato correttamente");
+        }
+
+        this.js().executeScript("arguments[0].setAttribute('autocomplete', 'off')" , emailBox);
+        emailBox.sendKeys(Keys.chord(Keys.CONTROL,"a"),Keys.DELETE);
```

Comment:
I don't understand what's the purpose of this action

Reply:
it turns off the autocomplete, as when we try to insert the email, autocomplete inserts an email as well.